### ISSUE TITLE
CircularProgressbarWithChildren does not center children

### ISSUE THREAD

**Issue by cdosborn** [kevinsqi/react-circular-progressbar#94]
### Summary of issue

I was able to correctly center text by setting the svg's root style `display: block`. I learned that the default display is inline. This causes the svg to have an expanded height that is greater than its set height to account for descenders in text that can appear alongside it. The absolute positioned children would be centered in the expanded height as opposed to the svg's visual height.

**Comment by kevinsqi** [comment]
hmm, you shouldn't need to set the svg to have `display: block` in order to center text. The text should already be centered in most modern browsers, and there's a known issue in IE with [two suggested workarounds](https://github.com/kevinsqi/react-circular-progressbar#fixing-text-centering-in-internet-explorer-ie).

If that doesn't address your issue, please add the following detail to this issue:

* What browser version you're on
* Create a [Codesandbox example](https://codesandbox.io/s/reactcircularprogressbar-issue-template-3zm3j) that reproduces the issue you're having, and take a screenshot of the non-centered text.

**Comment by vitorsilvalima** [comment]
it is not being centralised correctly as mention by @cdosborn 

<img width="202" alt="Screen Shot 2019-05-31 at 13 46 29" src="https://user-images.githubusercontent.com/9752658/58721131-8628a380-83aa-11e9-9c19-859b6b704dfb.png">

**Comment by kevinsqi** [comment]
Okay, thanks for the screenshot @vitorsilvalima - I think see the issue.  My mistake for not noticing it earlier - the reason I didn't see it is because I always include bootstrap 4 CSS in my demo apps, which includes a "reboot" file that contains the following declaration:

```css
svg {
  vertical-align: 'middle';
}
```

Adding that should fix the centering issue. So to work around the issue temporarily, you could add this style to the root svg element:

```jsx
<CircularProgressbarWithChildren
  value={20}
  styles={{
    root: {
      verticalAlign: "middle"
    }
  }}
>
  <div>...some children...</div>
</CircularProgressbarWithChildren>
```

This Codesandbox demonstrates the issue and shows this workaround fix: **https://codesandbox.io/s/i086m**

I'll also include a proper fix in the next release so you don't have to use this workaround, which I'll try to push out soon.

**Comment by kevinsqi** [comment]
This should now be fixed in v2.0.1 - thanks @cdosborn and @vitorsilvalima for reporting the issue! Let me know if you have any further issues.
